react/src/lib/components/Menu/types/size.ts

```diff
@@ -0,0 +1,4 @@
+export type Size = {
```

Comment:
Thought: Would it be of interest to have Size as a class with related utility functions in utils/geometry.ts as static methods?

Reply:
The upside of not implementing it as a class is the reduction of size of the packed productive JS file. The way it is now with several individually exported functions gives more flexibility and potentially helps holding the code size small since you only import what you really need and not the whole class.